cirq/sim/clifford/stabilizer_state_ch_form.py

```diff
@@ -73,18 +73,18 @@ def _json_dict_(self) -> Dict[str, Any]:
     def _from_json_dict_(cls, n, G, F, M, gamma, v, s, omega, **kwargs):
         copy = StabilizerStateChForm(n)
 
-        copy.G = G.copy()
-        copy.F = F.copy()
-        copy.M = M.copy()
-        copy.gamma = gamma.copy()
-        copy.v = v.copy()
-        copy.s = s.copy()
+        copy.G = np.array(G.copy())
```

Comment:
You can remove the `copy`. This can be just `np.array(G)` which copies by default. [otherwise, you should be using `np.asarray`, which doesn't copy]

Reply:
Thanks, I'll send out a follow up PR